q2_phylogeny/plugin_setup.py

```diff
@@ -50,8 +50,10 @@
     },
     parameter_descriptions={
         'n_threads': 'The number of threads. Using more than one thread '
-                     'runs the non-deterministic variant of `FastTree`, and '
-                     'may result in a different tree than single-threading. '
+                     'runs the non-deterministic variant of `FastTree` '
+                     '(`FastTreeMP`), and may result in a different tree than '
+                     'single-threading. See '
+                     'http://www.microbesonline.org/fasttree/ for details. '
```

Comment:
Do we want to include the anchor tag? `http://www.microbesonline.org/fasttree/#OpenMP`

Reply:
Awesome, I couldn't find an anchor! Updated the PR with another squash commit.